lms/djangoapps/bookmarks/views.py

```diff
@@ -0,0 +1,278 @@
+"""
+For more information, see:
```

Comment:
Can you also add "HTTP end-points for the Bookmarks API." at the top of the docstring?